### PR TITLE
MS. Laud Misc. 626: Peter Kidd description

### PR DESCRIPTION
**Pull request by holfordm**

### CODE REVIEW

File: collections/Laud_Misc/MS_Laud_Misc_626.xml [hunk 1]
```diff
@@ -39,20 +42,69 @@
...
                   <msItem n="1" xml:id="MS_Laud_Misc_626-item1">
+                     <locus>(fols. 1r–190v)</locus>
                      <author key="person_69105608">Hugutio of Pisa</author>
                      <title key="work_2175">Liber deriuationum</title>
+                     <msItem>
+                        <locus>(fol. 1r)</locus>
+                        <note>Prologue</note>
+                        <rubric>Incipit prologus super derivationes maiores</rubric>
+                        <incipit> Cum nostri protoplasti suggestiva prevaricatione. <sic>dyabolus</sic> humanum genus a sue dignitatis culmine quam longe <choice><sic>deiecerit</sic> <corr>deciderit</corr></choice> </incipit>
+                        <explicit>a verbo augmenti noftre assertionis auspicium sorciamur</explicit>
```
Comment (adunning) [inline collections/Laud_Misc/MS_Laud_Misc_626.xml:53]: noftre = nostre, presumably

File: collections/Laud_Misc/MS_Laud_Misc_626.xml [hunk 2]
```diff
@@ -39,20 +42,69 @@
...
+                        <finalRubric>Explicit prologus</finalRubric>
+                        <note>The author is referred to as ‘ … patria Pisanus nomine <sic>Huicio</sic> …’. </note>
```
Comment (adunning) [inline collections/Laud_Misc/MS_Laud_Misc_626.xml:55]: Is it possible to give a reference for this quotation?

File: collections/Laud_Misc/MS_Laud_Misc_626.xml [hunk 3]
```diff
@@ -39,20 +42,69 @@
...
+                     </msItem>
+                     <note>A leaf is lost (apparently excised) before fol. 1 (with loss of a table of contents(?)); for other missing leaves see below, Collation. Some of these losses are signalled in marginal pencil notes by a 19th- or early 20th-century century reader, not always correctly (e.g. there is no leaf missing after fol. 101).</note>
                      <textLang mainLang="la">Latin</textLang>
+                     <!-- text online: https://dama.dantenetwork.it/fileadmin/texts/Uguccione_Derivationes.xml
+                      = 
+                     https://dama.dantenetwork.it/index.php?id=17&workSign=Uguccione_Derivationes&L=0
+                     -->
+                  </msItem>
+                  <msItem>
+                     <note>One gloss in German, fol. 82r: <q>auis preda<damage/> scilicet valcke</q></note>
```
Comment (adunning) [inline collections/Laud_Misc/MS_Laud_Misc_626.xml:71]: Is `<gap/>` meant instead of `<damage/>`?

Comment (holfordm) [reply collections/Laud_Misc/MS_Laud_Misc_626.xml:71]: text lost due to cropping

File: collections/Laud_Misc/MS_Laud_Misc_626.xml (hunk 1 above)
Comment (holfordm) [reply collections/Laud_Misc/MS_Laud_Misc_626.xml:53]: !

File: collections/Laud_Misc/MS_Laud_Misc_626.xml (hunk 2 above)
Comment (holfordm) [reply collections/Laud_Misc/MS_Laud_Misc_626.xml:55]: good idea

**Pull request merged**